[Storage,Doc] Add additional info on bundling for IE11 (#8993)
Azure/azure-sdk-for-js: sdk/storage/storage-blob/README.md

### JavaScript Bundle

To use this client library in the browser, first you need to use a bundler. For details on how to do this, please refer to our [bundling documentation](https://aka.ms/AzureSDKBundling).

#### Special bundling notes for IE11

> Currently only `Parcel` and `Rollup` work well with Storage client libraries.

Reviewer: ```suggestion
Currently only `Parcel` and `Rollup` work well with Storage client libraries for IE11.
```

> If `Parcel` is used  then no further work is needed. Otherwise, an additional step is needed to transform the bundled output to the format that IE11 supports:

Reviewer: ```suggestion
If `Parcel` is used  then no further work is needed. If using Rollup, an additional step is needed to transform the bundled output to the format that IE11 supports.
```

> (Assuming `bundled-output.js` is the result from `Rollup`.)

Reviewer: ```suggestion
Assuming `bundled-output.js` is the result from `Rollup`:
```
